hal/lp_ticker_api.h

```diff
@@ -30,10 +30,37 @@ extern "C" {
 #endif
 
 /**
- * \defgroup hal_LpTicker Low Power Ticker Functions
+ * \defgroup hal_lp_ticker Low Power Ticker
+ * Low level interface to the low power ticker of a target
+ *
+ * # Defined behavior
```

Comment:
These specifications are great, but the rough outline should be repeated in the C++ classes so people know what performance to expect.